.travis.yml

```diff
@@ -14,26 +14,34 @@ addons:
 
 os:
     - linux
-    - osx
 
 compiler:
-    - clang
```

Comment:
Wouldn't keeping this and excluding linux+clang be better? So you don't need to duplicate the build configs for clang+osx.

Also, it might make sense to only keep gcc and clang here and also keep the `env` configs below, and then `include` gcc-5 and clang-3.6 builds as needed.